### ISSUE TITLE
Feature request: External trigger

### ISSUE THREAD

**Issue by Heidistein** [8go/matrix-eno-bot#13]
For some idea I am working on (API interaction) it is possible to issue a call and the response will eventually be POST'ed to a webserver. Have a system that not only responds to commands in channels, but also on an external trigger.
This could be as simple as a file appearing in a directory or a full-blown HTTP server.

Low priority, now solved by just POST'ing to the matrix server

**Comment by 8go** [comment]
Thanks @Heidistein for the additional details. Your input is food for thoughts.

As an alternative, available now, is to use `matrix-commander` from repo https://github.com/8go/matrix-commander. 

One can build about anything witht his CLI tool. `matrix-commander` can be used to send messages to rooms. It cculd be called from from `cron` (`crontab`), from a script that polls NFS volumes, etc.

**Comment by Heidistein** [comment]
Yes (diverting from the topic now a bit, sorry), that is a good one. I have looked at it. However I ended up just sending a HTTP PUT to the matrix server itself:
`curl -X PUT "https://${SERVER_HOST}/_matrix/client/r0/rooms/${ROOM_ID}/send/m.room.message/${RANDOM}" -H "Authorization: Bearer ${ACCESS_TOKEN}" -d "${CONTENT}"`
